### ISSUE TITLE
Unicode: UnicodeEncodeError: 'decimal' codec can't encode character u'\x00' in position 2

### ISSUE THREAD

**Issue by synctext** [Tribler/tribler#2859]
While running 7 .0 Beta on Ubuntu (part of the 26MByte log file).

```
ERROR:TorrentDBHandler:Could not create a TorrentDef instance '\xb6\x1b\xfa\xfeo\xf6\xfe:\xbe_A\x0f6\xd8\xac\xbfU\x0b\x08\x93' 1486457804 u'VODO.avi' ((u'VODO.avi', 736282624),) ('udp://tracker.1337x.org:80\x00',) {'dispersy_id': 651415}
Traceback (most recent call last):
  File "/usr/share/tribler/Tribler/Core/CacheDB/SqliteCacheDBHandler.py", line 345, in addExternalTorrentNoDef
    torrent_id = self._addTorrentToDB(torrentdef, extra_info)
  File "/usr/share/tribler/Tribler/Core/CacheDB/SqliteCacheDBHandler.py", line 431, in _addTorrentToDB
    self._addTorrentTracker(torrent_id, torrentdef, extra_info)
  File "/usr/share/tribler/Tribler/Core/CacheDB/SqliteCacheDBHandler.py", line 485, in _addTorrentTracker
    tracker_url = get_uniformed_tracker_url(announce)
  File "/usr/share/tribler/Tribler/Core/Utilities/tracker_utils.py", line 47, in get_uniformed_tracker_url
    if not url.port:
  File "/usr/lib/python2.7/urlparse.py", line 113, in port
    port = int(port, 10)
UnicodeEncodeError: 'decimal' codec can't encode character u'\x00' in position 2: invalid decimal Unicode string
```

**Comment by synctext** [comment]
note the URL ```udp://tracker.1337x.org:80\x00```, chokes on the trailing \x00.

**Comment by devos50** [comment]
Pretty sure this has been fixed by #2869.
